pallets/core/src/modules/trust_registry/impl.rs

```diff
@@ -74,12 +73,16 @@ impl<T: Config> Pallet<T> {
     }
 
     pub(super) fn update_delegated_issuers_(
```

Comment:
Can you please explain the what this function is doing, likely by adding comments in the body of function at various placed?